polaris-pipeline/Common/Services/SearchIndexService/SearchIndexService.cs

```diff
@@ -123,32 +123,34 @@ public async Task<bool> WaitForStoreResultsAsync(AnalyzeResults analyzeResults,
         {
             _logger.LogMethodEntry(correlationId, nameof(WaitForStoreResultsAsync), $"Wait for Search Indexation, CaseId={cmsCaseId}, DocumentId={cmsDocumentId}, Version={versionId}");
 
-            var sentLines = analyzeResults.ReadResults.SelectMany(r => r.Lines.Select(l => l.Text)).ToHashSet();
+            var sentLinesCount = analyzeResults.ReadResults.Sum(r => r.Lines.Count);
 
             var options = new SearchOptions
             {
                 Filter = $"caseId eq {cmsCaseId} and documentId eq '{cmsDocumentId}' and versionId eq {versionId}",
```

Comment:
I've seen (link below) that adding `Size = 0` means that we don't pull back the data.  Also that totalCount may not reach the exact count if there are additions and deletions in flux, but I guess we can always wait until it settles.  But what if it never does for some reason?
https://learn.microsoft.com/en-us/rest/api/searchservice/search-documents#query-parameters